src/patternfly/components/ProgressStepper/progress-stepper.scss

```diff
@@ -0,0 +1,253 @@
+    --pf-c-progress-stepper__step--m-current__step-title--FontWeight: var(--pf-c-progress-stepper--m-compact__step-title--FontWeight);
+
+    .pf-c-progress-stepper__step:not(.pf-m-current) .pf-c-progress-stepper__step-main {
+      @include pf-u-screen-reader;
+    }
+
+    .pf-c-progress-stepper__step-description {
+      display: none;
+    }
+
+    // For this compact variant only, move the main content for the current step to above the icons
+    // stylelint-disable
+    .pf-c-progress-stepper__step.pf-m-current .pf-c-progress-stepper__step-main {
+        grid-column: 1/-1;
+        grid-row: 1/2;
+      }
+    // stylelint-enable
+  }
+}
+
+// Step
+.pf-c-progress-stepper__step {
+  display: contents;
+
+  // Step Modifiers
+  &.pf-m-current {
+    --pf-c-progress-stepper__step-title--FontWeight: var(--pf-c-progress-stepper__step--m-current__step-title--FontWeight);
+    --pf-c-progress-stepper__step-title--Color: var(--pf-c-progress-stepper__step--m-current__step-title--Color);
+  }
+
+  &.pf-m-pending {
+    --pf-c-progress-stepper__step-title--Color: var(--pf-c-progress-stepper__step--m-pending__step-title--Color);
+  }
+
+  &.pf-m-success {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--success-color--100);
+  }
+
+  &.pf-m-danger {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--danger-color--100);
+    --pf-c-progress-stepper__step-title--Color: var(--pf-c-progress-stepper__step--m-danger__step-description--Color);
+  }
+
+  &.pf-m-warning {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--warning-color--100);
+  }
+
+  &.pf-m-info {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--info-color--100);
+  }
+}
+
+// The step icon wrapper provides a container on which to use the before for the line connecting the steps
+.pf-c-progress-stepper__step-connector {
+  position: relative;
+  display: flex;
+  justify-content: var(--pf-c-progress-stepper__step-connector--JustifyContent);
+  width: 100%;
+
+  .pf-c-progress-stepper__step:not(:last-of-type) > &::before {
+    position: absolute;
+    top: var(--pf-c-progress-stepper__step-connector--before--Top); // half the height
+    left: var(--pf-c-progress-stepper__step-connector--before--Left);
+    width: var(--pf-c-progress-stepper__step-connector--before--Width);
+    height: var(--pf-c-progress-stepper__step-connector--before--Height);
+    content: "";
+    border-right: var(--pf-c-progress-stepper__step-connector--before--BorderRightWidth) solid var(--pf-c-progress-stepper__step-connector--before--BorderRightColor);
+    border-bottom: var(--pf-c-progress-stepper__step-connector--before--BorderBottomWidth) solid var(--pf-c-progress-stepper__step-connector--before--BorderBottomColor);
+    transform: var(--pf-c-progress-stepper__step-connector--before--Transform);
+  }
+}
+
+// Step icon
+.pf-c-progress-stepper__step-icon {
+  z-index: var(--pf-global--ZIndex--sm);
```

Comment:
Oops, missed this - can you use the `--xs` var, and also make a var for this (`--pf-c-progress-stepper__step-icon--ZIndex: var(--pf-global--ZIndex--xs);`